### ISSUE TITLE
Bug: Updater ignores negative answer on quitting question

### ISSUE THREAD

**Issue by meiphoo7-Mae** [ferdium/ferdium-app#879]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.2.4-nightly.5

### What Operating System are you using?

Ubuntu

### Operating System Version

Ubuntu 22

### What arch are you using?

x64

### Last Known Working Ferdium version

Not sure. I assume it has always worked like this.

### Expected Behavior

Ferdium should respect the setting of the 'Confirm when quitting Ferdium' switch at all times even in the unlikely case that the user invokes the updater and answers 'no' to the question.

### Actual Behavior

When the 'Confirm when quitting Ferdium' switch is active and the user gives a negative answer to the question while the updater is invoked then the update is still performed (while it should be cancelled).
This leaves the tray icon visible but it is inactive because, I assume, it isn't connected to the running instance of Ferdium any more.

### Steps to reproduce

Please do as I described under Expected Behavior.

### Additional information

I tried this only once.
I know this is a corner case but sometimes I intentionally subject software to unusual situations in order to figure out if it behaves well in such situations.

After all the project may benefit from it  :grin:

**Comment by vraravam** [comment]
imo, the issue is not with the updater per se. Even without going through the update process, if you try to quit Ferdium (and the option to confirm is turned on), it will bring up the confirmation dialog. At this juncture, if you respond in the negative, I have noticed that the Ferdium window closes, while Activity Monitor still shows that Ferdium (the process) is still running.

**Comment by meiphoo7-Mae** [comment]
My issue is about that the updater still performs the update (and thus quits and restarts Ferdium) even when the user has decided to disallow quitting. I know this because the PID of Ferdium has changed.
What you describe is also true but I didn't test that until you brought it up.

**Comment by theAkito** [comment]
Same issue on Windows 11, but independent of updater. This just happens any time you click the X and then select "No" in the dialogue. Does this deserve a separate issue?
